tools/rosclean/src/rosclean/__init__.py

```diff
@@ -57,15 +68,21 @@ def _ask_and_call(cmds, cwd=None):
     def quote(s):
         return '"%s"'%s if ' ' in s else s
     sys.stdout.write("Okay to execute:\n\n%s\n(y/n)?\n"%('\n'.join([' '.join([quote(s) for s in c]) for c in cmds])))
-    while 1:
-        input = sys.stdin.readline().strip().lower()
-        if input in ['y', 'n']:
-            break
-    accepted = input == 'y'
+    accepted = _check_user_input()
```

Comment:
This could be changed to:

```
accepted = _ask('\n'.join([' '.join([quote(s) for s in c]) for c in cmds]))
```

Then `_check_user_input` doesn't need to be separated from `_ask` anymore.